borg/testsuite/archiver.py

```diff
@@ -1444,6 +1444,20 @@ def test_extra_chunks(self):
         self.cmd('check', self.repository_location, exit_code=0)
         self.cmd('extract', '--dry-run', self.repository_location + '::archive1', exit_code=0)
 
+    def test_attic013_acl_bug(self):
+        archive, repository = self.open_archive('archive1')
+        with repository:
+            manifest, key = Manifest.load(repository)
+            with Cache(repository, key, manifest) as cache:
+                archive = Archive(repository, key, manifest, '0.13', cache=cache, create=True)
+                archive.items_buffer.add({
+                    b'path': '1234',
+                    b'mtime': 0,
+                    b'acl': None
```

Comment:
also add a comment here that this was a invalid and unintended key due to that bug.